### PR TITLE
CBO-377:  From controls, translations & page semantics

### PR DESCRIPTION
**Pull request by cleargif**
#### What
Adding aria markup to almost all the radio and check box instances across the site

Other changes include:
- translations moved to the `en.yml`
- semantic layouts to apply the correct html patterns

#### Ticket

[CBO-377](https://dsdmoj.atlassian.net/browse/CBO-377)

#### Why

DAC report

#### How

In most cases the `fieldset > legend` + the `radio|checkbox label`  are combined to create the `aria-label` 
--------

### CODE REVIEW

File: lib/extensions/string_extension.rb [hunk 1]
```diff
@@ -39,6 +39,10 @@ def abbreviate(target_length = 6)
     acronym.upcase
   end
 
+  def to_legend
```
Comment (jsugarman) [inline lib/extensions/string_extension.rb]: **not a blocker** could we change this to either `to_slug`, `to_css_id`, `to_css_class` or something more generic as this method could possibly prove useful elsewhere.

Comment (cleargif) [reply lib/extensions/string_extension.rb]: 👍

File: app/views/external_users/claims/case_details/_retrial_detail_fields.html.haml [hunk 2]
```diff
@@ -9,16 +9,17 @@
   = f.adp_text_field :retrial_estimated_length, label: t('.retrial_estimated_length'), hint_text: t('.retrial_estimated_length_hint'), input_classes:'form-control-1-8', input_type: 'number', errors: @error_presenter
 
   = f.adp_text_field :retrial_actual_length, label: t('.retrial_actual_length'), hint_text: t('.retrial_actual_length_hint'), input_classes:'form-control-1-8', input_type: 'number', errors: @error_presenter
+  = "sdfsdfsdfsdf"
```
Comment (jsugarman) [inline app/views/external_users/claims/case_details/_retrial_detail_fields.html.haml]: is this some debug code left in?

**Review by jsugarman — APPROVED** [review]
🏆 looks great. Just a debug statement left in by the looks of it.

File: app/views/external_users/claims/case_details/_retrial_detail_fields.html.haml (hunk 2 above)
Comment (cleargif) [reply app/views/external_users/claims/case_details/_retrial_detail_fields.html.haml]: removed

**Pull request merged**